browser/brave_tab_helpers.cc

```diff
@@ -116,7 +118,9 @@ void AttachTabHelpers(content::WebContents* web_contents) {
 #endif
 
 #if BUILDFLAG(IPFS_ENABLED)
-  ipfs::IPFSTabHelper::MaybeCreateForWebContents(web_contents);
+  ipfs::IPFSTabHelper::MaybeCreateForWebContents(web_contents,
```

Comment:
formatting should go like this
```
  ipfs::IPFSTabHelper::MaybeCreateForWebContents(
      web_contents,
      brave::IsRegularProfile(web_contents->GetBrowserContext()),
      new ipfs::IpfsTabHelperDelegateImpl());
```

Reply:
I did clang-format on all IPFS files. I'll create an issue about our npm run lint also doing a clang-format to compare input and output and if there are mismatches to fail the lint for changed files.